obspy/io/rg16/__init__.py

```diff
@@ -7,18 +7,24 @@
 `Farfield Nodal <fairfieldnodal.com>`_'s
 `Zland <http://fairfieldnodal.com/equipment/zland>`_ product line.
 
+:author:
+    Derrick Chambers
+    Romain Pestourie (Ecole et Observatoire des Sciences de la Terre)
 :copyright:
-    The ObsPy Development Team (devs@obspy.org)
+    the ObsPy Development Team (devs@obspy.org)
 :license:
     GNU Lesser General Public License, Version 3
     (https://www.gnu.org/copyleft/lesser.html)
 
 .. note::
-    1. Some useful diagrams, provided by Faifield Nodal technical support,
```

Comment:
These aren't in the manual. I recommend keeping this link in the docs.

Reply:
I removed the link because the two first figures seem to me not very obvious to understand.
The third figure is mentioned later /_images/rg16_node_orientation.png.